packages/riipen-ui/src/components/RadioButtonGroup.jsx

```diff
@@ -47,8 +64,15 @@ const RadioButtonGroup = ({
     child =>
       child &&
       React.cloneElement(child, {
+        checked: value === child.props.value,
         onChange: handleChange,
-        classes: [linkedStyles.className, "radioButton"]
+        ...other,
+        ...child.props,
+        classes: [
+          linkedStyles.className,
+          "radioButton",
+          ...(child.props.classes || [])
+        ]
```

Comment:
With this we can just pass a "value" to the radio button group and it will set the correct child as checked